pkg/cvo/internal/operatorstatus.go

```diff
@@ -232,9 +235,10 @@ func waitForOperatorStatusToBeDone(ctx context.Context, interval time.Duration,
 			Nested: fmt.Errorf("cluster operator %s is not done; it is available=%v, progressing=%v, degraded=%v",
 				actual.Name, available, progressing, degraded,
 			),
-			Reason:  "ClusterOperatorNotAvailable",
-			Message: fmt.Sprintf("Cluster operator %s has not yet reported success", actual.Name),
-			Name:    actual.Name,
+			Progressing: progressing,
```

Comment:
This adds sensitivity to the `Progressing` condition, which [we should drop][1].  And maybe that's a reason to name your new property something other than `Progressing`?  Because the issue is less "are we attempting to reconcile divergent state" and more "are we worried about the current state".  E.g. running an outdated operator for a few minutes while it updates is progressing, but should not result in `Failing=True`.  And running a leveled, happy operator is not progressing, but should not result in `Failing=True` either.  Possibly call it `Satisfactory`?  Or call it `Failing` and make it a `*bool` to distinguish folks who aren't setting it?  I dunno...

[1]: https://github.com/openshift/cluster-version-operator/blame/b6cfb178a4cb0ddb735084244643f3bbd7a34c78/docs/user/reconciliation.md#L137